CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Thanks for taking the time to join our community and start contributing!
+
+Please remember to read and observe the [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+This  project accepts contribution via github [pull requests](https://help.github.com/articles/about-pull-requests/).
```

Comment:
Double space found here `This project`

Reply:
@SteveChurch thanks for catching that!

fixed. :+1: